### ISSUE TITLE
CSS "mix-blend-mode: plus-lighter" causes Lighthouse accessibility to fail

### ISSUE THREAD

**Issue by medeno123** [Leaflet/Leaflet#9699]
### Checklist

- [x] I've looked at the [documentation](https://leafletjs.com/reference.html) to make sure the behavior isn't documented and expected.
- [x] I'm sure this is an issue with Leaflet, not with my app or other dependencies (Angular, Cordova, React, etc.).
- [x] I've searched through the [current issues](https://github.com/Leaflet/Leaflet/issues) to make sure this hasn't been reported yet.
- [x] I agree to follow the [Code of Conduct](https://github.com/Leaflet/Leaflet/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.

### Steps to reproduce

Go to the leaflet website, on windows Chrome, run Lighthouse.  Then an exclamation mark shows for the acessibility score.  I noticed this problem for about a year now.  Comment out "mix-blend-mode: plus-lighter: and the exclamation mark goes away.  Not a problem with Leaflet, but only with Chrome Lighthouse evaluating Leaflet.

### Expected behavior

No exclamation mark when running Lighthouse

### Current behavior

![Image](https://github.com/user-attachments/assets/71e65b14-3204-46bc-9c32-8ba5d1bccc94)

### Minimal example reproducing the issue

https://leafletjs.com  then run Chrome Lighthouse

### Environment

- Leaflet version: 
- Browser (with version): 
- OS/Platform (with version):

**Comment by Falke-Design** [comment]
"Works as designed" because of: #3575
